diff --git a/node_modules/@atb-as/utils/lib/offers/ticket-offer.d.ts b/node_modules/@atb-as/utils/lib/offers/ticket-offer.d.ts
index 972e980..23ae696 100644
--- a/node_modules/@atb-as/utils/lib/offers/ticket-offer.d.ts
+++ b/node_modules/@atb-as/utils/lib/offers/ticket-offer.d.ts
@@ -59,6 +59,18 @@ export type TicketRoute = z.infer<typeof TicketRoute>;
 export declare const TicketOffer: z.ZodObject<{
     offerId: z.ZodString;
     travellerId: z.ZodString;
+    supplementProducts: z.ZodArray<z.ZodObject<{
+        id: z.ZodString;
+        price: z.ZodObject<{
+            originalAmount: z.ZodString;
+            originalAmountFloat: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodNumber>>, z.ZodTransform<number | undefined, number | null | undefined>>;
+            amount: z.ZodString;
+            amountFloat: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodNumber>>, z.ZodTransform<number | undefined, number | null | undefined>>;
+            currency: z.ZodString;
+            vatGroup: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
+        }, z.core.$strip>;
+        selectableId: z.ZodString;
+    }, z.core.$strip>>;
     price: z.ZodObject<{
         originalAmount: z.ZodString;
         originalAmountFloat: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodNumber>>, z.ZodTransform<number | undefined, number | null | undefined>>;
@@ -67,7 +79,7 @@ export declare const TicketOffer: z.ZodObject<{
         currency: z.ZodString;
         vatGroup: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
     }, z.core.$strip>;
-    fareProduct: z.ZodString;
+    fareProduct: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
     validFrom: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
     validTo: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
     flexDiscountLadder: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodObject<{
@@ -109,7 +121,7 @@ export declare const TicketOffers: z.ZodArray<z.ZodObject<{
         currency: z.ZodString;
         vatGroup: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
     }, z.core.$strip>;
-    fareProduct: z.ZodString;
+    fareProduct: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
     validFrom: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
     validTo: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodString>>, z.ZodTransform<string | undefined, string | null | undefined>>;
     flexDiscountLadder: z.ZodPipe<z.ZodOptional<z.ZodNullable<z.ZodObject<{
diff --git a/node_modules/@atb-as/utils/lib/offers/ticket-offer.js b/node_modules/@atb-as/utils/lib/offers/ticket-offer.js
index 1ab7235..ee3a8ba 100644
--- a/node_modules/@atb-as/utils/lib/offers/ticket-offer.js
+++ b/node_modules/@atb-as/utils/lib/offers/ticket-offer.js
@@ -57,8 +57,13 @@ exports.TicketRoute = v4_1.z.object({
 exports.TicketOffer = v4_1.z.object({
     offerId: v4_1.z.string(),
     travellerId: v4_1.z.string(),
+    supplementProducts: v4_1.z.array(v4_1.z.object({
+        id: v4_1.z.string(),
+        price: exports.SearchOfferPrice,
+        selectableId: v4_1.z.string(),
+    })),
     price: exports.SearchOfferPrice,
-    fareProduct: v4_1.z.string(),
+    fareProduct: v4_1.z.string().nullish().transform(utils_1.nullishToOptional),
     validFrom: v4_1.z.string().nullish().transform(utils_1.nullishToOptional),
     validTo: v4_1.z.string().nullish().transform(utils_1.nullishToOptional),
     flexDiscountLadder: exports.FlexDiscountLadder.nullish().transform(utils_1.nullishToOptional),
