diff --git a/node_modules/@entur-private/abt-token-state-react-native-lib/ios/Inspection/BarcodeInspectionModule.swift b/node_modules/@entur-private/abt-token-state-react-native-lib/ios/Inspection/BarcodeInspectionModule.swift
index 0ab1c52..5ee088c 100644
--- a/node_modules/@entur-private/abt-token-state-react-native-lib/ios/Inspection/BarcodeInspectionModule.swift
+++ b/node_modules/@entur-private/abt-token-state-react-native-lib/ios/Inspection/BarcodeInspectionModule.swift
@@ -13,6 +13,6 @@ class BarcodeInspectionModule: RCTViewManager {
     }
     
     @objc override static func requiresMainQueueSetup() -> Bool {
-        return true
+        return false
     }
 }
diff --git a/node_modules/@entur-private/abt-token-state-react-native-lib/ios/Inspection/VisualInspectionModule.swift b/node_modules/@entur-private/abt-token-state-react-native-lib/ios/Inspection/VisualInspectionModule.swift
index 3a73c04..ae2ea6f 100644
--- a/node_modules/@entur-private/abt-token-state-react-native-lib/ios/Inspection/VisualInspectionModule.swift
+++ b/node_modules/@entur-private/abt-token-state-react-native-lib/ios/Inspection/VisualInspectionModule.swift
@@ -13,7 +13,7 @@ class VisualInspectionModule: RCTViewManager {
     }
     
     @objc override static func requiresMainQueueSetup() -> Bool {
-        return true
+        return false
     }
     
 }
