diff --git a/node_modules/@react-native-mapbox-gl/maps/ios/RCTMGL/RCTMGLMapViewManager.m b/node_modules/@react-native-mapbox-gl/maps/ios/RCTMGL/RCTMGLMapViewManager.m
index c456ff5..df16797 100644
--- a/node_modules/@react-native-mapbox-gl/maps/ios/RCTMGL/RCTMGLMapViewManager.m
+++ b/node_modules/@react-native-mapbox-gl/maps/ios/RCTMGL/RCTMGLMapViewManager.m
@@ -490,7 +490,7 @@ - (void)mapViewRegionIsChanging:(MGLMapView *)mapView
 
 - (void)mapView:(MGLMapView *)mapView regionDidChangeWithReason:(MGLCameraChangeReason)reason animated:(BOOL)animated
 {    
-    if ((reason & MGLCameraChangeReasonTransitionCancelled) == MGLCameraChangeReasonTransitionCancelled) return;
+    if (((reason & MGLCameraChangeReasonTransitionCancelled) == MGLCameraChangeReasonTransitionCancelled) && ((reason & MGLCameraChangeReasonGesturePan) != MGLCameraChangeReasonGesturePan)) return;
 
     ((RCTMGLMapView *) mapView).isUserInteraction = (BOOL)(reason & ~MGLCameraChangeReasonProgrammatic);
     
