diff --git a/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.h b/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.h
index 24b09eb..20a04e0 100644
--- a/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.h
+++ b/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.h
@@ -62,6 +62,8 @@ NS_ASSUME_NONNULL_BEGIN
 // Interface for natively fetching all the keys from the storage data.
 - (void)getAllKeys:(RCTResponseSenderBlock)callback;
 
++ (void)setAppGroupName:(NSString *)groupName;
+
 @end
 
 NS_ASSUME_NONNULL_END
diff --git a/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm b/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
index fa3bb04..ef02130 100644
--- a/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
+++ b/node_modules/@react-native-async-storage/async-storage/ios/RNCAsyncStorage.mm
@@ -19,6 +19,7 @@
 static NSString *const RCTExpoStorageDirectory = @"RCTAsyncLocalStorage";
 static NSString *const RCTManifestFileName = @"manifest.json";
 static const NSUInteger RCTInlineValueThreshold = 1024;
+static NSString *AppGroupName;
 
 #pragma mark - Static helper functions
 
@@ -134,9 +135,14 @@ static void RCTAppendError(NSDictionary *error, NSMutableArray<NSDictionary *> *
     storageDirectoryPath =
         NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES).firstObject;
 #else
-    storageDirectoryPath =
+    if (!AppGroupName) {
+        storageDirectoryPath =
         NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES)
             .firstObject;
+    } else {
+        NSURL *pathUrl = [[NSFileManager defaultManager] containerURLForSecurityApplicationGroupIdentifier:AppGroupName];
+        storageDirectoryPath = pathUrl.path;
+    }
     // We should use the "Application Support/[bundleID]" folder for persistent data storage that's
     // hidden from users
     storageDirectoryPath = [storageDirectoryPath
@@ -665,6 +671,10 @@ - (BOOL)_passthroughDelegate
         [self.delegate respondsToSelector:@selector(isPassthrough)] && self.delegate.isPassthrough;
 }
 
++ (void)setAppGroupName:(NSString *)groupName {
+    AppGroupName = groupName;
+}
+
 #pragma mark - Exported JS Functions
 
 // clang-format off
