diff --git a/node_modules/@react-native-mapbox-gl/maps/index.d.ts b/node_modules/@react-native-mapbox-gl/maps/index.d.ts
index 58939cd..2d64b79 100644
--- a/node_modules/@react-native-mapbox-gl/maps/index.d.ts
+++ b/node_modules/@react-native-mapbox-gl/maps/index.d.ts
@@ -373,7 +373,7 @@ export interface MapViewProps extends ViewProps {
   logoEnabled?: boolean;
   compassEnabled?: boolean;
   compassViewPosition?: number;
-  compassViewMargins?: Point;
+  compassViewMargins?: {x: number; y:number};
   surfaceView?: boolean;
   regionWillChangeDebounceTime?: number;
   regionDidChangeDebounceTime?: number;
