diff --git a/node_modules/@react-native-mapbox-gl/maps/index.d.ts b/node_modules/@react-native-mapbox-gl/maps/index.d.ts
index 0f07d73..a4c34a2 100644
--- a/node_modules/@react-native-mapbox-gl/maps/index.d.ts
+++ b/node_modules/@react-native-mapbox-gl/maps/index.d.ts
@@ -398,7 +398,7 @@ export interface MapViewProps extends ViewProps {
   logoEnabled?: boolean;
   compassEnabled?: boolean;
   compassViewPosition?: number;
-  compassViewMargins?: Point;
+  compassViewMargins?: {x: number; y:number};
   surfaceView?: boolean;
   regionWillChangeDebounceTime?: number;
   regionDidChangeDebounceTime?: number;
