diff --git a/node_modules/@svgr/cli/dist/index.js b/node_modules/@svgr/cli/dist/index.js
index c004ad8..e61453c 100644
--- a/node_modules/@svgr/cli/dist/index.js
+++ b/node_modules/@svgr/cli/dist/index.js
@@ -222,7 +222,8 @@ const dirCommand = async (opts, _, filenames) => {
       if (!opts2.prettier)
         return fileContent;
       const prettierRcConfig = opts2.runtimeConfig ? await prettier$1.resolveConfig(filepath, { editorconfig: true }) : {};
-      return prettier$1.format(fileContent, __spreadValues$1(__spreadValues$1({}, prettierRcConfig), opts2.prettierConfig));
+      const prettierFileInfo = await prettier$1.getFileInfo(filepath);
+      return prettier$1.format(fileContent, __spreadValues$1(__spreadValues$1({ parser: prettierFileInfo.inferredParser }, prettierRcConfig), opts2.prettierConfig));
     })();
     await fs.promises.writeFile(filepath, prettyContent);
   };
