diff --git a/node_modules/react-native-dash/Dash.js b/node_modules/react-native-dash/Dash.js
index 32ff1ec..899bc60 100644
--- a/node_modules/react-native-dash/Dash.js
+++ b/node_modules/react-native-dash/Dash.js
@@ -5,15 +5,19 @@
 */
 
 import React from 'react'
-import PropTypes from 'prop-types'
-import { View, StyleSheet, ViewPropTypes } from 'react-native'
+import { View, StyleSheet } from 'react-native'
 import MeasureMeHOC from 'react-native-measureme'
 import { getDashStyle, isStyleRow } from '../util'
 
 const Dash = props => {
 	const isRow = isStyleRow(props.style)
-	const length = isRow ? props.width : props.height
-	const n = Math.ceil(length / (props.dashGap + props.dashLength))
+	let n;
+	if (props.dashCount == null) {
+		const length = isRow ? props.width : props.height
+		n = Math.ceil(length / (props.dashGap + props.dashLength))
+	} else {
+		n = props.dashCount
+	}
 	const calculatedDashStyles = getDashStyle(props)
 	let dash = []
 	for (let i = 0; i < n; i++) {
@@ -42,15 +46,6 @@ const styles = StyleSheet.create({
 	column: { flexDirection: 'column' },
 })
 
-Dash.propTypes = {
-	style: ViewPropTypes.style,
-	dashGap: PropTypes.number.isRequired,
-	dashLength: PropTypes.number.isRequired,
-	dashThickness: PropTypes.number.isRequired,
-	dashColor: PropTypes.string,
-	dashStyle: ViewPropTypes.style,
-}
-
 Dash.defaultProps = {
 	dashGap: 2,
 	dashLength: 4,
diff --git a/node_modules/react-native-dash/dist/index.js b/node_modules/react-native-dash/dist/index.js
index 4034e31..0f3e833 100644
--- a/node_modules/react-native-dash/dist/index.js
+++ b/node_modules/react-native-dash/dist/index.js
@@ -8,10 +8,6 @@ var _react = require('react');
 
 var _react2 = _interopRequireDefault(_react);
 
-var _propTypes = require('prop-types');
-
-var _propTypes2 = _interopRequireDefault(_propTypes);
-
 var _reactNative = require('react-native');
 
 var _reactNativeMeasureme = require('react-native-measureme');
@@ -24,8 +20,13 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 
 var Dash = function Dash(props) {
 	var isRow = (0, _util.isStyleRow)(props.style);
-	var length = isRow ? props.width : props.height;
-	var n = Math.ceil(length / (props.dashGap + props.dashLength));
+	var n;
+	if (props.dashCount == null) {
+		var length = isRow ? props.width : props.height;
+		n = Math.ceil(length / (props.dashGap + props.dashLength));
+	} else {
+		n = props.dashCount
+	}
 	var calculatedDashStyles = (0, _util.getDashStyle)(props);
 	var dash = [];
 	for (var i = 0; i < n; i++) {
@@ -53,15 +54,6 @@ var styles = _reactNative.StyleSheet.create({
 	column: { flexDirection: 'column' }
 });
 
-Dash.propTypes = {
-	style: _reactNative.ViewPropTypes.style,
-	dashGap: _propTypes2.default.number.isRequired,
-	dashLength: _propTypes2.default.number.isRequired,
-	dashThickness: _propTypes2.default.number.isRequired,
-	dashColor: _propTypes2.default.string,
-	dashStyle: _reactNative.ViewPropTypes.style
-};
-
 Dash.defaultProps = {
 	dashGap: 2,
 	dashLength: 4,
diff --git a/node_modules/react-native-dash/index.d.ts b/node_modules/react-native-dash/index.d.ts
index a234223..c7dd059 100644
--- a/node_modules/react-native-dash/index.d.ts
+++ b/node_modules/react-native-dash/index.d.ts
@@ -6,6 +6,7 @@ import React from 'react';
 import { ViewStyle, StyleProp } from 'react-native';
 
 export interface DashProps {
+  dashCount?: number;
   dashGap: number;
   dashLength: number;
   dashThickness: number;
