diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index 86bace7..9bf7b19 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -314,11 +314,12 @@ task prepareReanimatedHeadersForPrefabs(type: Copy) {
     into(reanimatedPrefabHeadersDir)
 }
 
-task cleanCmakeCache() {
-    tasks.getByName("clean").dependsOn(cleanCmakeCache)
-    doFirst {
-        delete "${projectDir}/.cxx"
-    }
+task cleanCmakeCache(type: Delete) {
+    delete layout.projectDirectory.dir(".cxx")
+}
+
+tasks.named("clean").configure {
+    dependsOn(cleanCmakeCache)
 }
 
 repositories {