diff --git a/node_modules/react-native/react.gradle b/node_modules/react-native/react.gradle
index 84b1f60..a4e30e3 100644
--- a/node_modules/react-native/react.gradle
+++ b/node_modules/react-native/react.gradle
@@ -123,6 +123,10 @@ afterEvaluate {
 
         def enableHermes = enableHermesForVariant(variant)
 
+        // Set up dev mode
+        def devEnabled = !(config."devDisabledIn${targetName}"
+            || targetName.toLowerCase().contains("release"))
+
         def currentBundleTask = tasks.create(
             name: "bundle${targetName}JsAndAssets",
             type: Exec) {
@@ -149,10 +153,6 @@ afterEvaluate {
             // Set up the call to the react-native cli
             workingDir(reactRoot)
 
-            // Set up dev mode
-            def devEnabled = !(config."devDisabledIn${targetName}"
-                || targetName.toLowerCase().contains("release"))
-
             def extraArgs = []
 
             if (bundleConfig) {
@@ -180,7 +180,7 @@ afterEvaluate {
                     def hermesFlags;
                     def hbcTempFile = file("${jsBundleFile}.hbc")
                     exec {
-                        if (targetName.toLowerCase().contains("release")) {
+                        if (!devEnabled) {
                             // Can't use ?: since that will also substitute valid empty lists
                             hermesFlags = config.hermesFlagsRelease
                             if (hermesFlags == null) hermesFlags = ["-O", "-output-source-map"]
@@ -321,7 +321,6 @@ afterEvaluate {
         // This should really be done by packaging all Hermes related libs into
         // two separate HermesDebug and HermesRelease AARs, but until then we'll
         // kludge it by deleting the .so files out of the /transforms/ directory.
-        def isRelease = targetName.toLowerCase().contains("release")
         def libDir = "$buildDir/intermediates/transforms/"
         def vmSelectionAction = {
             fileTree(libDir).matching {
@@ -329,7 +328,7 @@ afterEvaluate {
                     // For Hermes, delete all the libjsc* files
                     include "**/libjsc*.so"
 
-                    if (isRelease) {
+                    if (!devEnabled) {
                         // Reduce size by deleting the debugger/inspector
                         include '**/libhermes-inspector.so'
                         include '**/libhermes-executor-debug.so'
