diff --git a/node_modules/react-native-dash/Dash.js b/node_modules/react-native-dash/Dash.js
index 32ff1ec..42bbee1 100644
--- a/node_modules/react-native-dash/Dash.js
+++ b/node_modules/react-native-dash/Dash.js
@@ -12,8 +12,13 @@ import { getDashStyle, isStyleRow } from '../util'
 
 const Dash = props => {
 	const isRow = isStyleRow(props.style)
-	const length = isRow ? props.width : props.height
-	const n = Math.ceil(length / (props.dashGap + props.dashLength))
+	let n;
+	if (props.dashCount == null) {
+		const length = isRow ? props.width : props.height
+		n = Math.ceil(length / (props.dashGap + props.dashLength))
+	} else {
+		n = props.dashCount
+	}
 	const calculatedDashStyles = getDashStyle(props)
 	let dash = []
 	for (let i = 0; i < n; i++) {
diff --git a/node_modules/react-native-dash/dist/index.js b/node_modules/react-native-dash/dist/index.js
index 4034e31..b09a27f 100644
--- a/node_modules/react-native-dash/dist/index.js
+++ b/node_modules/react-native-dash/dist/index.js
@@ -24,8 +24,13 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 
 var Dash = function Dash(props) {
 	var isRow = (0, _util.isStyleRow)(props.style);
-	var length = isRow ? props.width : props.height;
-	var n = Math.ceil(length / (props.dashGap + props.dashLength));
+	var n;
+	if (props.dashCount == null) {
+		var length = isRow ? props.width : props.height;
+		n = Math.ceil(length / (props.dashGap + props.dashLength));
+	} else {
+		n = props.dashCount
+	}
 	var calculatedDashStyles = (0, _util.getDashStyle)(props);
 	var dash = [];
 	for (var i = 0; i < n; i++) {
@@ -55,6 +60,7 @@ var styles = _reactNative.StyleSheet.create({
 
 Dash.propTypes = {
 	style: _reactNative.ViewPropTypes.style,
+	dashCount: _propTypes2.default.number,
 	dashGap: _propTypes2.default.number.isRequired,
 	dashLength: _propTypes2.default.number.isRequired,
 	dashThickness: _propTypes2.default.number.isRequired,
diff --git a/node_modules/react-native-dash/index.d.ts b/node_modules/react-native-dash/index.d.ts
index a234223..c7dd059 100644
--- a/node_modules/react-native-dash/index.d.ts
+++ b/node_modules/react-native-dash/index.d.ts
@@ -6,6 +6,7 @@ import React from 'react';
 import { ViewStyle, StyleProp } from 'react-native';
 
 export interface DashProps {
+  dashCount?: number;
   dashGap: number;
   dashLength: number;
   dashThickness: number;
